doc/users/interactive.rst

```diff
@@ -240,8 +231,7 @@ Preserve aspect ratio              hold **CONTROL** when panning/zooming with mo
 Other Python prompts
 ====================
 
-If you can not or do not want to use IPython, interactive mode works
-in the vanilla python prompt
+Interactive mode works in the default Python prompt:
```

Comment:
There is no setting to change to _not_ make this work, I prefer the original wording.

Reply:
I'm confused 'cause I just removed the leading clause so the meaning doesn't actually change.

Reply:
sorry I miss read as "works by default in the Python prompt" which is a very different sentence.